platform.txt

```diff
@@ -25,7 +25,7 @@ compiler.sdk.path={runtime.platform.path}/tools/sdk
 
 compiler.c.cmd=xtensa-esp32-elf-gcc
-compiler.c.flags=-std=gnu99 -Os -g3 -Wno-frame-address -ffunction-sections -fdata-sections -fstrict-volatile-bitfields -mlongcalls -nostdlib -fstack-protector {compiler.warning_flags} -Wno-error=unused-function -Wno-error=unused-but-set-variable -Wno-error=unused-variable -Wno-error=deprecated-declarations -Wextra -Wno-unused-parameter -Wno-sign-compare -Wwrite-strings -Wno-old-style-declaration -mfix-esp32-psram-cache-issue -MMD -c
+compiler.c.flags=-std=gnu99 -Os -g3 -Wno-frame-address -ffunction-sections -fdata-sections -fstrict-volatile-bitfields -mlongcalls -nostdlib -fstack-protector {compiler.warning_flags} -Wno-error=unused-function -Wno-error=unused-but-set-variable -Wno-error=unused-variable -Wno-error=deprecated-declarations -Wextra -Wno-unused-parameter -Wno-sign-compare -Wwrite-strings -Wno-old-style-declaration -Wno-implicit-fallthrough -mfix-esp32-psram-cache-issue -MMD -c
```

Comment:
Why ignore more warnings?  Why not add some annotation to the location the warnings are in and leave it enabled to catch accidental fall-thru cases?  This blog entry has a thorough discussion of the annotation options: https://developers.redhat.com/blog/2017/03/10/wimplicit-fallthrough-in-gcc-7/